addons/mail/static/src/models/thread_needaction_preview_view.js

```diff
@@ -8,6 +8,23 @@ registerModel({
     name: 'ThreadNeedactionPreviewView',
     identifyingFields: ['notificationListViewOwner', 'thread'],
     recordMethods: {
+        /**
+         * @param {MouseEvent} ev
+         */
+        onClick(ev) {
+            if (!this.exists()) {
+                return;
+            }
+            const markAsRead = this.markAsReadRef.el;
+            if (markAsRead && markAsRead.contains(ev.target)) {
+                // handled in `_onClickMarkAsRead`
```

Comment:
Note that this comment should probably be updated at some point to match the new name of the handler (i.e. without the `_`)

Reply:
TBH I think we should not trust comment inside functions, and eventually remove them when the code is fully under control.

Here it's a bit weird to just "cancel" the handling, so comment can help understanding it, even when it doesn't exactly refer to name of method in this record.